src/app/map/main-map/main-map.component.ts

```diff
@@ -12,21 +16,47 @@ export class MainMapComponent implements OnInit, OnDestroy {
 
   // this is needed to auto-open the map disclaimer modal on page load
   @ViewChild('modal') modal: ElementRef;
+  @ViewChild('mineMap', {static: true}) mineMap;
+  @ViewChild('mapSearch', {static: true}) mapSearch;
   modalRef: NgbModalRef;
 
   @Input() animate = true;
   @Input() showBoundaries = true;
   @HostBinding('class.full-screen') fullScreen = true;
 
+  public _loading = false;
```

Comment:
The other components uses `loading` not sure if they should match.

Reply:
they should for consistency